mitosheet/mitosheet/save_paths.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+import os
+
+MITO_CONFIG_KEY_HOME_FOLDER = 'MITO_HOME_FOLDER'
```

Comment:
We should also make sure to give users an example of this. I originally added the .mito to the the end of the path I created because I thought I was specifying the entire path. 

We could call this MITO_CONFIG_PATH_TO_HOME_FOLDER_KEY, but I don't think thats any clearer really.

Reply:
I'll update all of them, because CONFIG_KEY should be grouped and the text after should be the key. 

Will update the text.